apps/frontend/src/shared-research/ResearchOutput.tsx

```diff
@@ -1,10 +1,25 @@
-import { NotFoundPage, SharedResearchOutput } from '@asap-hub/react-components';
+import {
+  NotFoundPage,
+  SharedResearchOutput,
+  SharedResearchOutputGrantDocument,
+} from '@asap-hub/react-components';
 import { sharedResearch, useRouteParams } from '@asap-hub/routing';
+import { ResearchOutputResponse } from '@asap-hub/model';
 
 import { useBackHref } from '../hooks';
 import { useResearchOutputById } from './state';
 import Frame from '../structure/Frame';
 
+const getResearchOutputTemplate = (type: ResearchOutputResponse['type']) => {
```

Comment:
Can we do something like it was before? https://github.com/yldio/asap-hub/pull/1150/files#diff-d532068a9bbf7564c5df0eae332d711fc99586a2084562d38e21e5e03ba71607L22. Its allot more readable

Reply:
IMO, we could keep it as only one research output component, since most of the things are going to be the same. with the container, header contact info and just have a component for the different part with is the description tags and access information which are the differences